### PR TITLE
[FEAT] Reply service

### PR DESCRIPTION
**Pull request by ThanatWongsamut**
- Reply service

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
 version: "3.8"
 
 services:
+    mongodb:
```
Comment (sarunnut1236) [inline docker-compose.yml:4]: Shall we use Atlas in the future?

File: reply-service/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Use the official Node.js image as the base image
+FROM node:16
```
Comment (sarunnut1236) [inline reply-service/Dockerfile:2]: Shall we use alpine ?

**Review by sarunnut1236 — COMMENTED** [review]
Shall we change the project to .ts?

File: reply-service/Dockerfile (hunk 2 above)
Comment (ThanatWongsamut) [reply reply-service/Dockerfile:2]: I think node is enough, don't need to be whole alpine

**Comment by ThanatWongsamut** [comment]
I think we need to merge first, then fix it later
- TS
- MongoDB

**Pull request merged**